### ISSUE TITLE
Skip to content button hard to access using Firefox

### ISSUE THREAD

**Issue by alexstine** [WordPress/openverse#2474]
## Description

When using Firefox, the Skip to content button seems to not be visible on the page until a focus event makes it appear. I am not sure why screen readers cannot find it as it works fine in Google Chrome, always visible and accessible.

## Reproduction

1. Enable a screen reader on Windows such as NVDA or JAWS.
2. Open https://openverse.org/ in Firefox.
3. Notice how the screen reader comes to the analytics banner as the first element, not the Skip to content button.
4. Compare with Google Chrome. Notice how opening https://openverse.org/ results in Skip to content being the first element on the page.

## Environment

- Device: Laptop.
- OS: Windows 10 Professional 64-bit
- Browser: Firefox.
- Version: 114.02.
- Other info: NVDA or JAWS screen reader.

## Additional context

I have a feeling this is related to Vue portals. React portals have varying support with assistive tech and I am sure Vue suffers from the same problem.

```
<div force-destroy="true" class="vue-portal-target" data-v-6fb81374=""><button type="button" class="group/button flex appearance-none items-center justify-center rounded-sm no-underline z-50 ms-2 mt-2 focus:fixed focus:absolute button_8qIVo filled-pink_TINkH size-medium_faRVV focus-slim-filled border skipButton_FKScN">Skip to content</button></div>
```

**Comment by zackkrida** [comment]
thanks for reporting this! We have plans to shift to a more straightforward approach of having the button always rendered in the html rather than dealing with portals, so hopefully we can fix this as part of that effort.
